src/daemon/rpc_command_executor.cpp

```diff
@@ -661,11 +661,18 @@ bool t_rpc_command_executor::print_connections() {
   for (auto & info : res.connections)
   {
     std::string address = info.incoming ? "INC " : "OUT ";
-    address += info.ip + ":" + info.port;
+    if (((epee::net_utils::address_type) info.address_type) == epee::net_utils::ipv6_network_address::get_type_id())
+    {
+      address += "[" + info.ip + "]:" + info.port;
+    }
+    else
+    {
+      address += info.ip + ":" + info.port;
+    }
```

Comment:
```suggestion
    address += info.address;
```

Also handles Tor/I2P addresses